doc/source/installation.rst

```diff
@@ -1,140 +1,209 @@
-We further recommend that you install and use ``mamba``, a faster drop-in replacement for ``conda``.
+`Mambaforge`_ is a minimal `conda`_ distribution that includes `mamba`_ by default, uses the `conda-forge`_ channel 
+(preferred for MDAnalysis) and avoids clutter from `Anaconda`_ base packages.
+
+To install MDAnalysis with `Mambaforge`_, first download the `mambaforge installer`_ and then run: 
 
 .. code-block:: bash
 
-    conda create --name mdanalysis
-    conda activate mdanalysis
-    conda install -c conda-forge mamba
+    mamba create -n mdanalysis mdanalysis
+    mamba activate mdanalysis
 
-To install the latest stable version of MDAnalysis via ``conda``, use the following command. This installs all dependencies needed for full analysis functionality (excluding external programs such as `HOLE`_):
+To install the :ref:`test suite <mdanalysistests>` (optional, ~90 MB):
 
 .. code-block:: bash
 
-    mamba install -c conda-forge mdanalysis
+    mamba install -c conda-forge MDAnalysisTests
+
+For more installation options (including Windows installers), see the `official mamba installation guide`_.
 
-To upgrade:
+To upgrade, use:
 
 .. code-block:: bash
 
     mamba update mdanalysis
 
-To install the tests:
+If you're using MDAnalysis in **JupyterLab**, install `ipywidgets`_ for progress bar support:
 
 .. code-block:: bash
 
-    mamba install -c conda-forge MDAnalysisTests
+    mamba install -c conda-forge ipywidgets
+
+**2. Alternative: Install mamba to an existing conda installation**
+
+If you already use `conda`_, you can add `mamba`_ to your base environment:
 
-If you intend to use MDAnalysis in JupyterLab, you will have to install
-an extra package for the progress bar in analysis classes:
+.. code-block:: bash
+
+    conda install -c conda-forge mamba
+
+Then create and activate an environment with MDAnalysis:
 
 .. code-block:: bash
 
-    conda install -c conda-forge nodejs
-    jupyter labextension install @jupyter-widgets/jupyterlab-manager
+    mamba create -n mdanalysis mdanalysis
+    mamba activate mdanalysis
+
+.. _pip-installation:
 
-pip
-=====
-The following command will install or upgrade the latest stable version of MDAnalysis via ``pip``, with core dependencies. This means that some packages required by specific analysis modules will not be installed.
+pip 
+===
+
+The following command will install or upgrade the latest stable version of MDAnalysis via `pip`_ with core dependencies. 
+This means that some packages required by specific analysis modules will not be installed.
 
 .. code-block:: bash
 
     pip install --upgrade MDAnalysis
 
-If you need to install a fully-featured MDAnalysis, add the ``analysis`` tag. As with ``conda``, this will not install external programs such as `HOLE`_.
+To install a fully-featured version of MDAnalysis (including optional analysis modules), use the ``analysis`` extra tag:
```

Comment:
There are additional tags available (see https://github.com/MDAnalysis/mdanalysis/blob/35d9d2e3ab08e7e6741b57fe02a7215fe3b91a6c/package/pyproject.toml#L69 )
* analysis: additional dependencies for analysis modules
* extra_formats: more file formats
* parallel: include dask as a dependency for parallel analysis beyond a single machine
* doc: all dependencies to build the documentation